server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsSupportPoundSymbol.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2025 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.usecases;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.eclipse.lsp.cobol.test.engine.UseCaseEngine;
+import org.junit.jupiter.api.Test;
+
+/** This test verifies that the British Pound Symbol doesn't cause problems in CICS */
+class TestCicsSupportPoundSymbol {
+
+  private static final String TEXT =
+      "       IDENTIFICATION DIVISION.\n"
+          + "       PROGRAM-ID. TEST1.\n"
+          + "       DATA DIVISION.\n"
+          + "       WORKING-STORAGE SECTION.\n"
+          + "       01 {$*VRB} PIC £(10)9.99-.\n"
+          + "       PROCEDURE DIVISION.\n"
+          + "            EXEC CICS \n"
+          + "            SEND MAP('POUND TESTING £')\n"
+          + "            MAPSET('POUND TESTING TWO £')\n"
```

Comment:
```suggestion
          + "            SEND MAP('POUND£')\n"
          + "            MAPSET('POUND£')\n"
```
to keep it semi-realistic - there is a size limit